docs/docson/build-schema.json

```diff
@@ -350,6 +350,19 @@
                 }
             }
         },
+        "react-specs": {
```

Comment:
I think this should be `jsx-spec` not `react-specs`.

Reply:
fixed https://github.com/rescript-lang/rescript-compiler/pull/5484/commits/81029b30f20e8b12b9dff1229228e31daa5e7c5b